hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/JsonMetadataRecordStoreMutationObserver.java

```diff
@@ -115,8 +117,9 @@ private void updateValueMetadataIfNeccessary(Record record, Object updateValue)
             } else {
                 valueMetadata = metadataInitializer.createFromObject(serializationService.toObject(updateValue));
             }
-        } catch (Throwable e) {
-            throw rethrow(e);
+        } catch (IOException e) {
+            // silently ignore exception. Json string is allowed to be invalid.
+            ignore(e);
```

Comment:
Minor: do we need to rethrow other exceptions?

Reply:
makes sense